project/AkkaBuild.scala

```diff
@@ -371,68 +371,68 @@ object AkkaBuild extends Build {
 
   lazy val samplesSettings = parentSettings ++ ActivatorDist.settings
 
```

Comment:
The build changes are only temporary, nothing in the build should be merged back to master.
However, we must get rid of those samples at some point, wasting so much time in starting the sbt prompt.